src/main/java/net/datafaker/formats/Json.java

```diff
@@ -280,6 +283,23 @@ private void value2String(T record, Object value, StringBuilder sb) {
                 sb.append("\"");
             }
         }
+
+        private List<T> getValuesFrom(FakeSequence<T> sequence) {
+            if (sequence == null) {
+                return null;
+            }
+
+            if (sequence.isInfinite()) {
+                throw new IllegalArgumentException("Trying to generate a CSV from infinite sequence");
```

Comment:
I think it's better when exception message says what should be done to fix the issue, e.g.
`The sequence should be finite`

Reply:
Yep, sounds reasonable, done.